docs/pages/style-sherpa.md

```diff
@@ -95,24 +95,27 @@ The boilerplate uses Foundation's Vertical Menu&mdash;one item is made for each
 
 {{{{raw}}}}
 ```handlebars
-<div class="column row"><div class="row collapse">
-  <div class="large-3 medium-4 columns" data-sticky-container>
-    <ul class="vertical menu">
+<div class="row">
```

Comment:
Shouldn't this also use the new grid?

Reply:
The example was derived from style sherpa, so the grid needs to be changed on that repo first.

Reply:
Ah ok. The rest seems to be fine.